docs/admin/kubeadm-config.md

```diff
@@ -0,0 +1,36 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm config
+notitle: true
+---
+{% capture overview %}
+# Kubeadm config
+Beginning with v1.8.0, kubeadm uploads the configuration of your cluster to a ConfigMap called 
+`kubeadm-config` in the `kube-system` namespace, and later reads that configuration when upgrading 
+in order to configure system components correctly. This provides a seamless user experience.
+
+With `kubeadm config view` you can view the above ConfigMap, or, if you initialized your cluster
+ using kubeadm v1.7.x or lower, you can use the `kubeadm config upload` commands to create this
+ConfigMap before executing `kubeadm upgrade`.
+
```

Comment:
Suggested change:
You can use `kubeadm config view` to view the ConfigMap. If you initialized your cluster using kubeadm v1.7.x or lower, you can use `kubeadm config upload` to create a ConfigMap before you execute `kubeadm upgrade`.